drivers/i2s/i2s_mcux_sai.c

```diff
@@ -333,13 +434,14 @@ static void enable_mclk_direction(const struct device *dev, bool dir)
 	const struct i2s_mcux_config *dev_cfg = dev->config;
 	uint32_t offset = dev_cfg->mclk_pin_offset;
 	uint32_t mask = dev_cfg->mclk_pin_mask;
-	uint32_t *gpr =  (uint32_t *)DT_REG_ADDR(DT_NODELABEL(iomuxcgpr)) + offset;
+	uint32_t *gpr = (uint32_t *)DT_REG_ADDR(DT_NODELABEL(iomuxcgpr)) + offset;
```

Comment:
This is not producing the correct result. Offset is the raw offset (in bytes), but the order of operations is adding `offset` uint32_t's, so the resultant gpr register is off by 4x.
On the RT1060 this results in register 0x400ac010 being written, when it should be 0x400ac004

Should be: 
```suggestion
	uint32_t *gpr = (uint32_t *)(DT_REG_ADDR(DT_NODELABEL(iomuxcgpr)) + offset);
```

Reply:
updated, thanks @NickolasLapp